Sources/SKCore/BuildSystemManager.swift

```diff
@@ -154,6 +154,18 @@ extension BuildSystemManager {
       .first
   }
 
+  /// Returns the target's module name as parsed from the `ConfiguredTarget`'s compiler arguments.
+  public func moduleName(for document: DocumentURI, in target: ConfiguredTarget) async -> String? {
+    guard let buildSettings = await buildSettings(for: document, in: target, language: .swift),
```

Comment:
Is it OK that the language is always .swift? Should it be a parameter? If so, how can I determine the language when scanning for tests?

Reply:
You should be able to use `self.defaultLanguage(for: document)`.